src/main/java/edu/wpi/grip/ui/ExceptionView.java

```diff
@@ -0,0 +1,191 @@
+package edu.wpi.grip.ui;
+
+import javafx.application.HostServices;
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.scene.control.*;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Priority;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Displays an alert with an exception and provides a place for a user to supply information about what caused the error.
+ * Includes a textbox with formatted markdown to allow the issue to be pasted into GitHub easily.
+ * Also, provides links with quick access to the githup issue page.
+ */
+public final class ExceptionView extends Alert {
+    private static final String PROJECT_ISSUE_LINK = "https://github.com/WPIRoboticsProjects/GRIP/issues";
+    private static final String ISSUE_PROMPT_QUESTION = "What were the actions that caused this error occurred?";
+
+    private static final String systemOptions[] = {
+            "javafx.version",
+            "java.runtime.name",
+            "java.vm.version",
+            "java.vm.vendor",
+            "java.vm.name",
+            "java.runtime.version",
+            "java.awt.graphicsenv",
+            "javafx.runtime.version",
+            "os.name",
+            "os.version",
+            "os.arch",
+            "file.encoding",
+            "java.vm.info",
+            "java.version",
+            "sun.arch.data.model",
+            "sun.cpu.endian"
+    };
+
+    private final String exceptionMessage;
+    private final String systemInfoMessage;
+    private final Throwable initialCause;
+
+    private final ButtonType copyToClipboardBtnType = new ButtonType("Copy To Clipboard");
+    private final ButtonType openGitHubIssuesBtnType = new ButtonType("Open GitHub Issues");
+
+    /**
+     * @see <a href="http://code.makery.ch/blog/javafx-dialogs-official/">Inspiration</a>
+     * @param throwable The throwable exception to display this alert for.
+     * @param services The host services, allows access to {@link HostServices#showDocument(String)} in order to display
+     *                 the issue website.
+     */
+    public ExceptionView(final Throwable throwable, final HostServices services) {
```

Comment:
Is there a reason you didn't use fxml for this?  I think that having hundreds of lines of java code to set styles will just make maintaining the gui a lot harder